src/ApplicationInsights.ServiceFabric.Native/Net45/CorrelatingRemotingMessageHandler.cs

```diff
@@ -100,11 +98,7 @@ private void InitializeCommonFields()
             // Create and prepare activity and RequestTelemetry objects to track this request.
             RequestTelemetry rt = new RequestTelemetry();
 
-            if (messageHeaders.TryGetHeaderValue(ServiceRemotingLoggingStrings.ParentIdHeaderName, out string parentId))
-            {
-                rt.Context.Operation.ParentId = parentId;
-                rt.Context.Operation.Id = GetOperationId(parentId);
-            }
+            string parentId = messageHeaders.TryGetHeaderValue(ServiceRemotingLoggingStrings.ParentIdHeaderName, out parentId) ? parentId : null;
```

Comment:
nit: declaration expression could be an option here

`messageHeaders.TryGetHeaderValue(ServiceRemotingLoggingStrings.ParentIdHeaderName, out string parentId)`